Update/some_010.txt

```diff
@@ -165,14 +165,14 @@ void main()
 //r鼻先数ミリのところで思いっきり吼えられて、私は後ろにのけぞる。…椅子から転げ落ちなかっただけでも、たぶん相当な幸運だった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "鼻先数ミリのところで思いっきり吼えられて、私は後ろにのけぞる。…椅子から転げ落ちなかっただけでも、たぶん相当な幸運だった。",
-			NULL, "She ambushed me a few millimeters from the tip of my nose, and I slipped back. ...I somehow avoided falling out of my chair. It was probably just luck.", Line_Normal);
+			NULL, "She shouted right in my face, making me throw my head back. ...By pure dumb luck, I managed to avoid falling out of my chair.", Line_Normal);
```

Comment:
```suggestion
			NULL, "She shouted right in my face, causing me to flinch backwards. ...By pure dumb luck, I managed to avoid falling out of my chair.", Line_Normal);
```